FluidNC/src/Maslow/Maslow.cpp

```diff
@@ -705,7 +710,10 @@ void Maslow_::safety_control() {
 }
 // Maslow main loop
 void Maslow_::update(){
-
+    if(error){
+        digitalWrite(REDLED, HIGH);
```

Comment:
This can't work, right? It gets randomly turned off again on line 733, right? Does it work for you?

Reply:
Ahh I see it returns first, nice. My bad